Areas/Identity/Data/ApplicationDbContext.cs

```diff
@@ -27,5 +27,7 @@ public void Configure(EntityTypeBuilder<ApplicationUser> builder)
     {
         builder.Property(u => u.FirstName).HasMaxLength(255);
         builder.Property(u => u.LastName).HasMaxLength(255);
+        builder.Property(u => u.Email).HasMaxLength(255);
+        builder.Property(u => u.Password).HasMaxLength(255);
```

Comment:
Same as in the application user class. We don't need that because is already defined by IdenityUser class (built-in class of .net mvc).